source/d3d8to9.cpp

```diff
@@ -15,6 +15,123 @@ PFN_D3DXLoadSurfaceFromSurface D3DXLoadSurfaceFromSurface = nullptr;
 std::ofstream LOG;
 #endif
 
+extern "C" HRESULT WINAPI ValidatePixelShader(const DWORD* pPixelShader, const D3DCAPS8* pCaps, BOOL ReturnErrors, char** pErrorsString)
+{
+	UNREFERENCED_PARAMETER(pCaps);
+	UNREFERENCED_PARAMETER(ReturnErrors);
```

Comment:
I guess you don't need these two lines anymore.

Reply:
Yep, caught them after I pushed :+1: .